mcpjam-inspector/.env.local

```
@@ -1,7 +1,7 @@
-VITE_CONVEX_URL=https://energized-ant-201.convex.cloud
-CONVEX_URL=https://energized-ant-201.convex.cloud
-VITE_WORKOS_CLIENT_ID=client_01KTN2EWHHJCKRB8RSR307X4SG
+VITE_CONVEX_URL=https://outstanding-fennec-304.convex.cloud
+CONVEX_URL=https://outstanding-fennec-304.convex.cloud
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONVEX_URL key should go before the VITE_CONVEX_URL key

(UnorderedKey)